### PR TITLE
Updated roles to use snafu as a package

### PR DESCRIPTION
**Pull request by amitsagtani97**

### CODE REVIEW

**Comment by rsevilla87** [comment]
@amitsagtani97, can you rebase this one, please?

File: roles/load-ycsb/tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,73 @@
+---
```
Comment (aakarshg) [inline roles/load-ycsb/tasks/main.yml:1]: delete this file?

File: roles/ycsb-bench/tasks/main.yml [hunk 2]
```diff
@@ -0,0 +1,73 @@
+---
```
Comment (aakarshg) [inline roles/ycsb-bench/tasks/main.yml:1]: this can also be deleted

**Pull request merged**